action.php

```diff
@@ -423,20 +428,13 @@ protected function generatePDF($cachefile, $event)
         }
 
         //some shortcuts to export settings
-        $hasToC = $this->getExportConfig('hasToC');
-        $levels = $this->getExportConfig('levels');
         $isDebug = $this->getExportConfig('isDebug');
-        $watermark = $this->getExportConfig('watermark');
 
-        // initialize PDF library
-        require_once(__DIR__ . "/DokuPDF.class.php");
+        $config = new Config($this->conf, $this->getDocumentLanguage($this->list[0]));
+        $mpdf = new DokuPDF($config);
+        $template = new Template($this->getConf('template'), $this->getConf('qrcodescale'));
```

Comment:
Only template from config setting. I intend to make it modifiable via BookCreator.